peps/pep-0790.rst

```diff
@@ -36,41 +36,61 @@ Motivation
 ==========
 
 :pep:`508` established environment markers to specify dependencies based on
-environment conditions.
-Since its adoption in 2015, the Python ecosystem has evolved and new features
-warrant their own environment markers.
+environment conditions. Since its adoption in 2015, the Python ecosystem has
+evolved and new features warrant their own environment markers.
 
-One such feature is the introduction of Python interpreters with the Global
-Interpreter Lock (GIL) disabled, which allows for free threading
-[#python-free-threading]_.
+The primary motivation for extending the allowed set of environment markers now
+is free-threaded CPython [#python-free-threading]_ emerging. Currently there is
+no way to distinguish between a GIL-enabled and a free-threaded CPython interpreter
+with an environment marker. This leads to real world issues for the adoption of
+free-threading and its incremental rollout. When a Python package is being made
+compatible with free-threaded CPython, it also needs all its build and runtime
+dependencies to be compatible. Capturing the first version of a dependency that
+is compatible precisely in metadata is currently not possible, and increasing
+the minimum version of a dependency also for the GIL-enabled build is usually
+undesirable since it unnecessarily limits compatibility between packages.
 
-The lack of environment markers leads to real world issues, some of which have
-been discussed in `Environment marker for free-threading`_, for example:
+
```

Comment:
Is the extra newline intended here?

Reply:
I think probably not. Will address in my branch after merge.